airbyte_cdk/sources/declarative/retrievers/simple_retriever.py

```diff
@@ -477,9 +477,9 @@ def _extract_record(stream_data: StreamData, stream_slice: StreamSlice) -> Optio
             # Record is not part of `StreamData` but is the most common implementation of `Mapping[str, Any]` which is part of `StreamData`
             return stream_data
         elif isinstance(stream_data, (dict, Mapping)):
-            return Record(dict(stream_data), stream_slice)
+            return Record(data=dict(stream_data), associated_slice=stream_slice)
```

Comment:
Should we pass `stream_name` here?

Reply:
I guess yes, added

Reply:
fixed in https://github.com/airbytehq/airbyte-python-cdk/pull/62/commits/7ab7e06df0a23ad5262cc73716f3413f498ff898;
mypy is much cleaner now